src/Tribe/Events/Attendee_List_Display.php

```diff
@@ -0,0 +1,231 @@
+<?php
+/**
+ * Class that detects if given post
+ * is displaying the Attendee List.
+ */
+
+namespace Tribe\Tickets\Events;
+
+use Exception;
+use InvalidArgumentException;
+use Tribe__Tickets__Main;
+use Tribe__Tickets_Plus__Attendees_List;
+use WP_Post;
+
+/**
+ * Class Attendee_List_Display
+ *
+ * @since TBD
+ *
+ * @package Tribe\Tickets\Events
+ */
+class Attendee_List_Display {
+
+	/**
+	 * @since TBD
+	 * @var string Meta name to control whether the Attendee List
+	 *             meta was changed by a shortcode in the content.
+	 */
+	protected static $attendee_list_by_shortcode = 'tribe_tickets_attendee_list_triggered_by_shortcode';
+
+	/**
+	 * Check if given event is hiding the attendees list.
+	 * @since TBD
+	 *
```

Comment:
:no_entry_sign: There must be exactly one blank line before the tags in a doc comment

<sub>hash: 9bbd1269626b800dcc57f536e1e82946</sub>